src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,10 @@
+@mixin positioning {
+  display: grid;
+  justify-content: center;
+}
+
+@mixin mediaQueries ($columns, $brake-point) {
+  @media screen and (min-width: $brake-point) {
+    grid-template-columns: repeat($columns, $card-width);
+  }
+}
```

Comment:
Nice mixin, but it's not named properly

Reply:
For example `columnTemplates` or something like this

Reply:
got it